packager/getDevToolsMiddleware.js

```diff
@@ -29,20 +41,19 @@ module.exports = function(options) {
         'If you still need this, please let us know.'
       );
     } else if (req.url === '/launch-chrome-devtools') {
+      if (isDebuggerConnected()) {
+        // Dev tools are already open; no need to open another session
+        res.end('OK');
+        return;
+      }
       var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
-      var script = 'launchChromeDevTools.applescript';
       console.log('Launching Dev Tools...');
-      execFile(
-        path.join(__dirname, script), [debuggerURL],
-        function(err, stdout, stderr) {
-          if (err) {
-            console.log('Failed to run ' + script, err);
-          }
-          console.log(stdout);
-          console.warn(stderr);
+      opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
```

Comment:
This causes iOS request to time out and it sends another one, which can open Chrome tab when you are not expecting it.